lib/kafka/ssl_socket_with_timeout.rb

```diff
@@ -163,6 +163,10 @@ def closed?
       @tcp_socket.closed? || @ssl_socket.closed?
     end
 
+    def to_io
```

Comment:
@tcrayford that would be lovely! I'm actually thinking about running the CI suite against a more stable cluster...